cdap-app-templates/cdap-etl/cdap-etl-core/src/test/java/co/cask/cdap/templates/etl/batch/DefaultManifestConfigurer.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.templates.etl.batch;
```

Comment:
True. Should be part of the adapter-template management code. I will remove this and pass in the ETLConfig object (which will be created as per the other comment on this PR).